src/CruxComponent.tsx

```diff
@@ -425,7 +425,7 @@ export class CruxComponentCreator {
                                                         <div style={{ display: "inline-block", width: "80%" }}>
                                                             <input type="text"
                                                                 style={{ width: "100%" }}
-                                                                value={(this.state.filterModel || {})[field.search.key]}
+                                                                value={this.state.filterModel && this.state.filterModel[field.search.key] || ""}
                                                                 onKeyPress={this.handleSearchKeyPress}
```

Comment:
It was throwing error, changing uncontrolled input to controlled input.
Reason is value is getting undefined. For input it expect empty string for controlled input